packages/flame/lib/src/cache/images.dart

```diff
@@ -49,6 +49,14 @@ class Images {
     _assets[name] = _ImageAsset.fromImage(image);
   }
 
+  /// Transform the base64 encoded image into an [Image] and adds it into the
+  /// cache.
+  Future<void> addBase64EncodedImage(String name, String base64Data) async {
```

Comment:
```suggestion
  Future<void> addFromBase64Data(String name, String base64Data) async {
```
Maybe? That it's an image should already be implicit from the cache name